spec/Dynamic-Validator.md

```diff
@@ -265,7 +265,7 @@ fn reprioritize(candidates: &mut Vec<Candidate>, target: &Address) {
 }
 ```
 
-### on TermEnd events
+On `TermEnd` event, the candidates will be reordered with the following algorithm.
```

Comment:
On the `TermEnd` event, candidates will be reordered with the following algorithm.